.gitignore

```diff
@@ -99,3 +99,6 @@ sklearn/metrics/_pairwise_distances_reduction/_middle_term_computer.pxd
 sklearn/metrics/_pairwise_distances_reduction/_middle_term_computer.pyx
 sklearn/metrics/_pairwise_distances_reduction/_radius_neighbors.pxd
 sklearn/metrics/_pairwise_distances_reduction/_radius_neighbors.pyx
+sklearn/neighbors/_ball_tree.pyx
+sklearn/neighbors/_binary_tree.pxi
+sklearn/neighbors/_kd_tree.pyx
```

Comment:
Those also need to be included to `setup.cfg`:

https://github.com/scikit-learn/scikit-learn/blob/e3d1f9ac39e4bf0f31430e779acc50fb05fe1b64/setup.cfg#L80-L101